libexec/scoop-bucket.ps1

```diff
@@ -45,16 +45,14 @@ function list_buckets {
             $updated = (Get-Item "$source\bucket").LastWriteTime | Get-Date -Format 'yyyy-MM-dd HH:mm:ss'
             $source = friendly_path $source
         }
-
         $buckets += New-Object PSObject -Property @{
             Name      = $bucket
             Source    = $source
             Updated   = $updated
             Manifests = $manifests
         }
     }
-
-    return $buckets | Select-Object Name, Source, Updated, Manifests
+    return ($buckets | Select-Object Name, Source, Updated, Manifests | Out-String).Trim()
```

Comment:
Why out-string? It is better to return PSObject, for manipulating pipeline (i.e. `scoop bucket list | select manifests`)

Reply:
wanted to remove blank lines

Reply:
Oh. I'm afraid there's no way to do that. PowerShell adds newlines when printing objects.

Reply:
3 blank lines after output in PS 5.1
2 blank lines after output in PS 7.2